pkg/utils/test/framework.go

```diff
@@ -507,6 +498,24 @@ func (f Framework) WithPowerMonitorAdditionalConfigMaps(configMapNames []string)
 	}
 }
 
+func (f Framework) CreateConfigMap(configMapName, namespace, config string) *corev1.ConfigMap {
```

Comment:
```suggestion
func  NewAdditionalKeplerConfigMap(configMapName, namespace, config string) *corev1.ConfigMap {
```

This function does not 'create' a configmap in the classical sense, similar to how the Create method is often used in frameworks to specifically create a Kubernetes resource